src/bin/pgcopydb/ld_stream.h

```diff
@@ -116,17 +116,30 @@ typedef struct LogicalMessageValues
 	LogicalMessageValue *array; /* malloc'ed area */
 } LogicalMessageValues;
 
+
 typedef struct LogicalMessageValuesArray
 {
 	int count;
 	int capacity;
 	LogicalMessageValues *array; /* malloc'ed area */
 } LogicalMessageValuesArray;
 
+typedef struct LogicalMessageAttribute
+{
+	char *name; /* malloc'ed area */
```

Comment:
Maybe too generic? If that's only going to host column/attribute names, maybe it could be `char *attname`?

Reply:
@dimitri Sure. I've updated PR as per your suggestion.